content/about/_index.md

```diff
@@ -4,19 +4,34 @@ layout = "index"
 weight = 1
 +++
 
-Welcome to the official **osu!uwaterloo** club website! The UW *osu!* club welcomes anyone interested in the game, regardless of their skill level, what modes they play, or however else they may choose to enjoy *osu!* (ahem* mappers). The club Discord is open to everyone, even if they are not a student attending the University of Waterloo.
+Welcome to **osu!UWaterloo**!
 
-osu!uwaterloo hosts a number of *osu!*-related events every term. This includes the trimesterly club tournament: Waterloo osu! Cup. Apart from that, there may be other special gameplay tournaments, trivia nights, *osu!* Skribbl.io, or whatever else the club members wish to organize. No term is the same, but you can always expect some fun events to be planned!
+osu!UWaterloo welcomes anyone interested in [osu!](https://osu.ppy.sh) regardless of skill level, gamemode, or however else they may choose to enjoy osu! We run a number of events each term  including:
+- Waterloo osu! Cup
+- osu!-related contests
+- Trivia
+- In-person meetups
 
-## Club Meetings
+No term is the same, but you can always expect some fun events to be planned!
+
+![A UWaterloo student playing osu!](/images/perpet.jpg)
 
-osu!uwaterloo meets in-person roughly once every two weeks. Specific times and dates are subject to change due to the nature of student life.
+## How to Join
+1. Join the [Discord Server](https://discord.gg/mfWqAFg).
+2. Fill out out the [Verification Form](https://forms.gle/Q1HUK8ywaSmMvcCT6).
+3. Ping the Club Executive role in the `#verification` channel.
+4. Participate in events!
 
-During the COVID-19 pandemic, club meetings were cancelled so as to protect the health of club members and other Waterloo students. In the event that another global health crisis emerges to once-more threaten societal stability, in-person meetings may be cancelled again.
+## Club Meetings
+
+osu!UWaterloo officially meets in-person bi-weekly, usually on Saturday 2-6pm at [PHYS 145](https://maps.app.goo.gl/eYAM98mDbcBJEHG8A). Please check the announcements channel in the Discord server to stay up-to-date!
```

Comment:
maybe we can style "announcements" similar to how `#verification` was done:

"announcements" -> "`#announcements`"